networkit/cpp/randomization/CurveballUniformTradeGenerator.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * CurveballUniformTradeGenerator.h
+ *
+ *  Created on: Jul 11, 2017
+ *      Author: Hung Tran <htran@ae.cs.uni-frankfurt.de>
+ */
+#ifndef RANDOMIZATION_CURVEBALL_UNIFORM_TRADE_GENERATOR_H
+#define RANDOMIZATION_CURVEBALL_UNIFORM_TRADE_GENERATOR_H
+
+#include "../Globals.h"
+
+namespace NetworKit {
+
+class CurveballUniformTradeGenerator {
+public:
+	using value_type = std::vector<std::pair<node, node> >;
```

Comment:
value_type is very generic. Can we use some more specific name? Also, I don't think _type has precedent in NetworKit. I would like the suffix to be dropped.

Reply:
We commonly use this type in our implementations mimicking STL's conventions. The generic name is by design and since it's the same for all generators, transformers and containers you do not think about how the type is called when chaining together templates. In this case it's not really used. So if you do not like it, I suggest completely dropping the type alias.